testcontainers/src/lib.rs

```diff
@@ -28,13 +28,15 @@
 //! # Ecosystem
 //!
 //! The testcontainers ecosystem is split into multiple crates, however, the `testcontainers` crate itself is a meta-crate, re-exporting the others. Usually, depending on `testcontainers` should be sufficient for most users needs.
```

Comment:
This is actually outdated, we only have a single crate now :)

Reply:
Sure, you're right.

Updated, please take a look if it looks good to you